pkg/apis/cluster/v1alpha1/types.go

```diff
@@ -215,14 +296,35 @@ type ResourceSummary struct {
 	// Total amount of allocatable resources on all nodes.
 	// +optional
 	Allocatable corev1.ResourceList `json:"allocatable,omitempty"`
+
 	// Allocating represents the resources of a cluster that are pending for scheduling.
 	// Total amount of required resources of all Pods that are waiting for scheduling.
 	// +optional
 	Allocating corev1.ResourceList `json:"allocating,omitempty"`
+
 	// Allocated represents the resources of a cluster that have been scheduled.
 	// Total amount of required resources of all Pods that have been scheduled to nodes.
 	// +optional
 	Allocated corev1.ResourceList `json:"allocated,omitempty"`
+
+	// AllocatableModeling represents the number of each resources modeling in a cluster that are available for scheduling.
```

Comment:
typo: resources --> resource

> represents the number of each resources modeling

The number of nodes xxxx?

Reply:
the number of each resource modeling in a cluster. Isn't it right?